dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/SourceCodeRepoInterface.java

```diff
@@ -205,7 +206,7 @@ public Service initializeService(String repositoryId) {
      * @return workflow with associated workflow versions
      */
     public abstract Workflow setupWorkflowVersions(String repositoryId, Workflow workflow, Optional<Workflow> existingWorkflow,
-            Map<String, WorkflowVersion> existingDefaults, Optional<String> versionName);
+            Map<String, WorkflowVersion> existingDefaults, Optional<String> versionName, boolean hardRefresh);
```

Comment:
Was like this before but abstract and interface methods should really have javadoc (filled in)

Reply:
+1
Was going to do documentation changes if we decided to go through with it.